### ISSUE TITLE
CustomCompoundBondForce does not have method setTabulatedFunctionParameters()

### ISSUE THREAD

**Issue by xqding** [openmm/openmm#1887]
According to the [Python and C++ API documentation](http://docs.openmm.org/latest/api-python/generated/simtk.openmm.openmm.CustomCompoundBondForce.html#simtk.openmm.openmm.CustomCompoundBondForce.setFunctionParameters), the class CustomCompoundBondForce should have method setTabulatedFunctionParameters for setting parameters if the function is not Continuous1DFunction, but the method is not defined for the class [CustomCompoundBondForce](https://github.com/pandegroup/openmm/blob/master/openmmapi/include/openmm/CustomCompoundBondForce.h).

**Comment by peastman** [comment]
Thanks for catching that.  The comment is incorrect.  What it should say is that you call `setFunctionParameters()` on the TabulatedFunction object.

**Comment by xqding** [comment]
Thanks for pointing this out, but it still does not solve my problem. 

I tried to change the parameters for a Continuous2DFunction in a CompoundBondForce. The way I do it now is calling `func = CompoundBondForce.getTabulatedFunction(0)`, which will return the reference to the TabulatedFunction object. A related question is how to call `setFunctionParameters()` on the TabulatedFunction object in python. The TabulatedFunction class is inherited by the Continuos2DFunction class, but the class TabulatedFunction itself does not have the method `setFunctionParameters()`. Therefore, the TabulatedFunction object has to be downcasted into the class Contious2DFunction in python. We can do this in C++. Can we do the downcasting in python? I tried to change the object's `__class__` attr, but it did not work.
